downloads/swan-lake-release-notes/swan-lake-2201.11.0/RELEASE_NOTE.md

```diff
@@ -48,23 +48,32 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 11 (2201.11.0)
 
 ## Ballerina library updates
 
-### Features
+### New features
 
 #### `http` package
 
-- Added static code rules 
-- Added relax data binding support for service and client data binding
+- Added relaxed binding support for service and client data binding. This provides the flexibility to bind nil values to optional fields and absent values to nilable fields.
+
+  ```ballerina
+  // Enable relaxed data binding on the client side.
+  http:Client httpClient = check new("http://localhost:9090", laxDataBinding = true);
+  
+  // Enable relaxed data binding on the server side.
+  @http:ServiceConfig {laxDataBinding: true}
+  service /api on new http:Listener(9090) {
+  }
+  ```
 
 ### Improvements
 
 #### `http` package
 
-- Added `anydata` support for `setPayload` methods in the request and response objects
-- Improved `@http:Query` annotation to overwrite the query parameter name in client
-- Improved `@http:Query` annotation to overwrite the query parameter name in service
-- Added header name mapping support in record fields
-- Migrated client and service data binding lang utils usage into `data.jsondata` module utils `toJson` and `parserAsType`
-- Added support for configuring server name to be used in the SSL SNI extension
+- Added `anydata` support for `setPayload` methods in the request and response objects.
+- Improved the `@http:Query` annotation to overwrite query parameter names in clients.
+- Improved the `@http:Query` annotation to overwrite query parameter names in services.
+- Added header name mapping support in record fields.
+- Migrated client and service data binding to use the `toJson` and `parserAsType` functions from the  `ballerina/data.jsondata` module instead of the `fromJsonWithType` function from the `ballerina.lang.value` module. This allows integrating the `jsondata:Name` annotation in the JSON to Ballerina record conversion.
```

Comment:
```suggestion
- Migrated client and service data binding to use the `toJson` and `parserAsType` functions from the  `ballerina/data.jsondata` module instead of the `fromJsonWithType` function from the `ballerina.lang.value` module. This change improves how JSON data is converted to Ballerina records and vice versa, by allowing field name overwriting using the `jsondata:Name` annotation.
```

Reply:
@MaryamZi Is this ok?

Reply:
```suggestion
- Migrated client and service data binding to use the `toJson` and `parserAsType` functions from the  `ballerina/data.jsondata` module instead of the `fromJsonWithType` function from the `ballerina.lang.value` module. This change improves how JSON data is converted to Ballerina records and vice versa, by allowing field names to be overridden using the `jsondata:Name` annotation.
```